internal/releasegen/team.go

```diff
@@ -54,6 +55,18 @@ func (t *Team) Process() error {
 		t.Details.Repos = append(t.Details.Repos, lpRepos...)
 	}
 
+	// Iterate over the Gitea orgs.
+	for _, org := range t.config.GiteaConfig {
+		log.Printf("processing gitea org: %s\n", org.Org)
+
+		odRepos, err := gitea.FetchOrgRepos(org)
```

Comment:
What does "od" mean? This should probably be `giteaRepos` or just `repos`.

Reply:
Ah, that's from when I had named the package `opendev` and had a bunch of `od` prefixes like the `gt` ones that got removed. Missed this one twice! Thanks, will fix.

`repos` by itself I would have liked to use in a bunch of places, but it then shadows the `repos` package.